test/e2e/subscription_e2e_test.go

```diff
@@ -2813,48 +2818,58 @@ func checkDeploymentWithPodConfiguration(t GinkgoTInterface, client operatorclie
 		return
 	}
 
-	check := func(container *corev1.Container) {
+	check := func(container *corev1.Container) error {
 		for _, e := range envVar {
 			existing, found := findEnvVar(container.Env, e.Name)
-			require.Truef(t, found, "env variable name=%s not injected", e.Name)
-			require.NotNil(t, existing)
-			require.Equalf(t, e.Value, existing.Value, "env variable value does not match %s=%s", e.Name, e.Value)
```

Comment:
general comment (as it also applies to the code changes made further down): do we risk losing debug information by removing the debug line when an assertion fails?

Reply:
I restore some of the error messages.  Thanks!